traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
I'm probably missing something here. Why a person login with ldap should be "root" tenant?
Please additionally note that I cannot guarantee the ID of the root tenant. Furthermore, a root tenant is a tenant with a null parent. There is no limitation currently forcing a single root tenant.